devito/visitors.py

```diff
@@ -143,6 +149,51 @@ def always_rebuild(self, o, *args, **kwargs):
         return o._rebuild(*new_ops, **okwargs)
 
 
+class PrintAST(Visitor):
```

Comment:
Yes, in fact we need to also pick up generic `cgen.Generable` objects here to make the `Block` printing nice. I'll add that...